README.md

```diff
@@ -156,10 +156,10 @@ HOF-bootstrap accepts the following options so a developer can customise element
 
 ### Options
 
-- `views`: Location of the base views relative to the root of your project. Defaults to 'views'. Set `views` to `false` if not present.
+- `views`: Location of the base views relative to the root of your project. Defaults to `__dirname + '/views'`.
+- `fields`: Location of the common fields relative to the root of your project. Defaults to `__dirname + '/fields'`.
+- `translations`: Location of the common translations relative to the root of your project. Defaults to `./translations`.
```

Comment:
the views in hof-template-partials are always used implicitly, this views option on the root level is to provide a directory of common views shared between different routes in the same app.